frontend/src/App.tsx

```diff
@@ -50,59 +52,61 @@ const App = (): React.ReactElement => {
       <SampleContextDispatcherContext.Provider
         value={dispatchSampleContextUpdate}
       >
-        <AuthContext.Provider
-          value={{ authenticatedUser, setAuthenticatedUser }}
-        >
-          <ChakraProvider theme={defaultTheme}>
-            <Router>
-              <Switch>
-                <Route exact path={Routes.LOGIN_PAGE} component={Login} />
-                <Route exact path={Routes.SIGNUP_PAGE} component={Signup} />
-                <PrivateRoute
-                  exact
-                  path={Routes.HOME_PAGE}
-                  component={Default}
-                />
-                <PrivateRoute
-                  exact
-                  path={Routes.CREATE_ENTITY_PAGE}
-                  component={CreatePage}
-                />
-                <PrivateRoute
-                  exact
-                  path={Routes.UPDATE_ENTITY_PAGE}
-                  component={UpdatePage}
-                />
-                <PrivateRoute
-                  exact
-                  path={Routes.DISPLAY_ENTITY_PAGE}
-                  component={DisplayPage}
-                />
-                <PrivateRoute
-                  exact
-                  path={Routes.EDIT_TEAM_PAGE}
-                  component={EditTeamInfoPage}
-                />
-                <PrivateRoute
-                  exact
-                  path={Routes.HOOKS_PAGE}
-                  component={HooksDemo}
-                />
-                <PrivateRoute
-                  exact
-                  path={Routes.ADMIN_DASHBOARD_PAGE}
-                  component={AdminDasboard}
-                />
-                <PrivateRoute
-                  exact
-                  path={`${Routes.ADMIN_MODULE_EDITOR_BASE_ROUTE}/:courseID/:moduleID`}
-                  component={ModuleEditor}
-                />
-                <Route exact path="*" component={NotFound} />
-              </Switch>
-            </Router>
-          </ChakraProvider>
-        </AuthContext.Provider>
+        <ApolloProvider client={client}>
+          <AuthContext.Provider
+            value={{ authenticatedUser, setAuthenticatedUser }}
+          >
+            <ChakraProvider theme={defaultTheme}>
+              <Router>
+                <Switch>
+                  <Route exact path={Routes.LOGIN_PAGE} component={Login} />
+                  <Route exact path={Routes.SIGNUP_PAGE} component={Signup} />
+                  <PrivateRoute
+                    exact
```

Comment:
not something we need to implement right now, but in the future if we wanted to make the homepage unauthenticated, is it just changing the route? also how are the different home pages for clients/admins expressed?